### ISSUE TITLE
Convert log file from yesterday to the new format during the upgrade, and add a preflight check

### ISSUE THREAD

**Issue by bozana** [pkp/pkp-lib#8804]
Additionally to the today's usage stats log file also consider the yesterday's usage stats log file in the upgrade -- convert it to the new format too.
Also add a pre-flight check if there are some older log files than yesterday's (in the folder 'usageEventLogs') and if there are some older log files that were not processed successfully (and thus 'stuck' in the folder 'processing', 'reject' or 'stage').

S. https://github.com/pkp/pkp-lib/issues/8014#issuecomment-1465867710

**Comment by bozana** [comment]
@NateWr, could you take a look at the PR above?

**Comment by bozana** [comment]
Ah, @NateWr, now I know why I did not consider yesterday's log file earlier -- the new code considers installation date as start for the new log files... So, maybe I will need to consider installation date - 1 day... Let me think about it and check... So, please wait with the PR review.

**Comment by NateWr** [comment]
Makes sense @bozana. Maybe you can save the date of the new log files, depending on which day is processed, in `site_settings` as part of the `I8508_ConvertCurrentLogFile`? I left a couple suggestions for wording of the error messages.

**Comment by bozana** [comment]
@NateWr, would you like to take a look at the second commit, that should considered your suggestions... ?

**Comment by bozana** [comment]
Thanks a lot @NateWr!
